docs/static/Developers.md

```diff
@@ -75,6 +75,57 @@ npm run dev-examples
 
 - Run a watched routine [devExamples.js](../../bin/devExamples.js) with [nodemon](https://www.npmjs.com/package/nodemon).
 
+### Develop with modified ud-viz library in your demo 
+
+When developing an application that uses 
+[UD-Viz](https://github.com/VCityTeam/UD-Viz)
+as library, you might need to propose modifications/fixes/improvements
+of UD-Viz. 
+In doing so you will need a set up that allows for simultaneously modifying 
+both codes (the one of your application and the one of UD-Viz) in order to
+co-evolve UD-Viz and its usage within your application. 
+
+In order to establish that specific developing context you might
+- clone both repositories side by side (that is your cloned application 
+  repository stands in the same directory as the cloned UD-Viz) on your local 
+- Modify your the `package.json` of your application in order to "point" 
+  to the cloned version of the UD-Viz library.
+  Entries of the form
+  ```json
+  "dependencies": {
+     "@ud-viz/*": "x.x.x"
+  }
+  ```
+  should replaced with
+  ```json
+  "dependencies": {
+    "@ud-viz/*": "file:../../../UD-Viz/packages/*"
```

Comment:
Are we sure to need that many `..` ? As newbie I was expecting `"file../UD-Viz/packages/*".

Reply:
It's true there are too many .. 😶